packages/treasury/src/stablecoinMachine.js

```diff
@@ -235,22 +228,6 @@ export async function start(zcf) {
     return zcf.makeInvitation(makeLoanHook, 'make a loan');
   }
 
-  // this overarching SCM holds ownershipTokens in the individual per-type
-  // vaultManagers
-
-  // one exposed (but closely held) method is to add a brand new collateral
```

Comment:
This information needs to be preserved. It's approximately design doc.

Reply:
I moved it to the top comment and reworded somewhat. Lemme know if you want to say it better.